dotnetv3/Cognito/scenarios/Cognito_Basics/Cognito_MVP/CognitoBasics.cs

```diff
@@ -0,0 +1,103 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// snippet-start:[cognito.dotnetv3.CognitoBasics.Main]
+
+// Before running this SDK for .NET (v3) code example, set up your development environment, including your credentials.
```

Comment:
SDK for .NET (v3) --> AWS SDK for .NET (v3)

Reply:
fixed